app/assets/sass/application.scss

```diff
@@ -14,6 +14,10 @@ $path: "/public/images/";
   position: absolute !important;
 }
 
+
+table.export-buttons td a:visited {
```

Comment:
Should this class be renamed to export-button and be applied to the <a> element directly.

Reply:
Does the export button under caseload tab also need this class?